src/app/pages/dashboard/components/practice-end-to-end-card/practice-end-to-end-card.html

```diff
@@ -1,12 +1,12 @@
 <button clear (click)="navigateToFakeJournal()">
-  <ion-card class="practice-mode-card">
+  <ion-card class="practice-mode-card" id="practice-mode-full-card">
     <ion-grid>
       <ion-row>
         <ion-col col-36 col-lg-26 align-self-center>
           <img src="assets/imgs/dashboard/practice-mode-full.png" alt="Practice marking a full test report" class="icon-practice-test-full"/>
         </ion-col>
         <ion-col col-58 col-lg-44 align-self-center class="button-col">
-          <ion-button class="mes-neutral-button">
+          <ion-button class="mes-neutral-button" id="practice-mode-full">
```

Comment:
same with this one, call this "practice-mode-full-button"

Reply:
done